server/Dockerfile.server

```diff
@@ -29,11 +29,19 @@ RUN apt-get update -y; \
     ca-certificates \
     curl \
     redis-tools \
+    unzip \
     ;
 
 RUN curl -fsSLO https://github.com/subtrace/subtrace/releases/download/b143/subtrace-linux-amd64 \
     && chmod +x ./subtrace-linux-amd64
 
+RUN curl -L -o libtorch-cxx11-abi-shared-with-deps-2.4.0+cpu.zip https://download.pytorch.org/libtorch/cpu/libtorch-cxx11-abi-shared-with-deps-2.4.0%2Bcpu.zip \
```

Comment:
This might also need to be located when compiling depenencies.

https://github.com/devflowinc/trieve/pull/2915/files#diff-0092f086be905d516d3457f02246497a74ff3594a5716321fcd13b17b31f9763R15

Reply:
The dependency is located using the envs below